src/app/components/ReclaimEscrowForm/index.tsx

```diff
@@ -76,8 +89,8 @@ export const ReclaimEscrowForm = memo((props: Props) => {
               placeholder={t('common.amount')}
               type="number"
               step="any"
-              min={0.0001}
-              max={Number(props.maxAmount) / 10 ** 9}
+              min="0"
```

Comment:
Is there some minimum amount for ReclaimEscrow?

Reply:
https://github.com/oasisprotocol/oasis-core/blob/master/go/consensus/tendermint/apps/staking/transactions.go#L333 not seeing any